IssueCreator/IssueCreator/Models/IssueMilestone.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace IssueCreator.Models
+{
+    public class IssueMilestone
+    {
+        public int? Number { get; private set; }
+        public string Title { get; private set; }
+        public DateTimeOffset? DueOn { get; private set; }
+
+        public static IssueMilestone FromGitHubMilestone(Octokit.Milestone m)
```

Comment:
Nit - Krzysztof would ask why this isn't just a .ctor.  He'd wait for you to explain, and then tell you it should just be a .ctor.